src/init.cpp

```diff
@@ -1763,7 +1763,7 @@ bool AppInit2(const std::vector<std::string>& words)
 
         if (!zwalletMain->GetMasterSeed().IsNull()) {
             //Inititalize zPIVWallet
-            uiInterface.InitMessage(_("Syncing zPIV wallet..."));
+            uiInterface.InitMessage(_("Syncing zZNZ wallet..."));
 
             //Load zerocoin mint hashes to memory
             pwalletMain->zpivTracker->Init();
```

Comment:
Should zpivTracker be zZNZTracker?

Reply:
Damn, not sure how I missed this, could lead to compiling errors, will resolve.